.github/workflows/rust.yml

```diff
@@ -22,6 +22,9 @@ jobs:
           profile: minimal
           toolchain: stable
           override: true
+      - uses: abelfodil/protoc-action@v1
```

Comment:
just for curiosity, why is this new step needed now?

Reply:
https://github.com/hyperium/tonic/issues/1047#issuecomment-1222508191